yt_dlp/extractor/novaplay.py

```diff
@@ -1,49 +1,62 @@
     _TESTS = [
         {
-            'url': 'https://play.nova.bg/video/bratya/season-3/bratq-2021-10-08/548677',
-            'md5': 'b1127a84e61bed1632b7c2ca9cbb4153',
+            'url': 'https://play.nova.bg/video/ochakvaite/season-0/ochakvaite-2022-07-22-sybudi-se-sat/606627',
+            'md5': 'd79dff2d09d196c595a7290f48e33399',
             'info_dict': {
-                'id': '548677',
+                'id': '606627',
                 'ext': 'mp4',
-                'title': 'Братя',
-                'alt_title': 'bratya/season-3/bratq-2021-10-08',
-                'duration': 1603.0,
-                'timestamp': 1633724150,
-                'upload_date': '20211008',
-                'thumbnail': 'https://nbg-img.fite.tv/img/548677_460x260.jpg',
-                'description': 'Сезон 3 Епизод 25'
+                'title': 'Събуди се - събота по NOVA (23.07.2022)',
+                'alt_title': 'ochakvaite/season-0/ochakvaite-2022-07-22-sybudi-se-sat',
+                'duration': 29.0,
+                'timestamp': 1658491547,
+                'upload_date': '20220722',
+                'thumbnail': 'https://nbg-img.fite.tv/img/606627_460x260.jpg',
+                'description': '29 сек',
+                'view_count': False
             },
         },
         {
-            'url': 'https://play.nova.bg/video/igri-na-volqta/season-3/igri-na-volqta-2021-09-20-1/548227',
-            'md5': '5fd61b8ecbe582fc021019d570965d58',
+            'url': 'https://play.nova.bg/video/ochakvaite/season-0/ochakvaite-2022-07-22-cherry-tazi/606609',
+            'md5': 'f3e973e2ed1a5b9b3f498b1ab82d01b3',
             'info_dict': {
-                'id': '548227',
+                'id': '606609',
                 'ext': 'mp4',
-                'title': 'Игри на волята: България (20.09.2021) - част 1',
-                'alt_title': 'gri-na-volqta/season-3/igri-na-volqta-2021-09-20-1',
-                'duration': 4060.0,
-                'timestamp': 1632167564,
-                'upload_date': '20210920',
-                'thumbnail': 'https://nbg-img.fite.tv/img/548227_460x260.jpg',
-                'description': 'Сезон 3 Епизод 13'
+                'title': 'Черешката на тортата - тази вечер по NOVA (22.07.2022)',
+                'alt_title': 'ochakvaite/season-0/ochakvaite-2022-07-22-cherry-tazi',
+                'duration': 29.0,
+                'timestamp': 1658476303,
+                'upload_date': '20220722',
+                'thumbnail': 'https://nbg-img.fite.tv/img/606609_460x260.jpg',
+                'description': '29 сек',
+                'view_count': False
             },
         }
     ]
 
+    def _get_access_token(self):
+        req = sanitized_Request('https://play.nova.bg/')
+        cookie = '; '.join(urllib.request.urlopen(req).info().get_all('Set-Cookie'))
+        return self._download_json('https://play.nova.bg/api/client', None, headers={
+            'cookie': cookie
+        })['accessToken']
```

Comment:
No need, the underlying network handling when using `_download_webpage` etc. will handle cookies for you.

Pretty sure this is redundant anyways. We will get these cookies from the webpage download below. So you should be able to move this access token download to after that and it should work.

Reply:
I had a feeling there is a better way to do it, thanks.